arviz/plots/forestplot.py

```diff
@@ -96,6 +96,24 @@ def plot_forest(
     Returns
     -------
     gridspec : matplotlib GridSpec
+
+    Examples
```

Comment:
These are great! Are you planning on adding examples for some of the other options/arguments as well?

Reply:
Hi! I just added a forestplot example (before it was only ridgeplot) and one more jointplot.

Reply:
Sounds good. Feel free to disregard my comment :)